### DISCUSSION TITLE
Database logs

### DISCUSSION THREAD

**Discussion by kenn7575** [ItzNimaB/sop-it-service-solution#7] [General]
What logs do we need in the database?

**Comment by HaZZaKeY** [comment]
Jeg tænkte det kunne være noget i den her stil:
![image](https://github.com/kenn7575/sop-it-service-solution/assets/117901485/7bf93fd5-b956-4226-8a35-ca5e3242c6bf)

Her med database info
![image](https://github.com/kenn7575/sop-it-service-solution/assets/117901485/d47e27d4-45f3-42ce-b7a3-df6797673b17)
